src/app/go-mobile/go-mobile.component.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, OnInit } from '@angular/core';
+import { GoMobileService } from './go-mobile.service';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { UtilsService, ContactNumberFormat } from '@services/utils.service';
+import { NotificationService } from '@shared/notification/notification.service';
+import { BrowserStorageService } from '@services/storage.service';
+import { environment } from '../../environments/environment.prod';
+
+@Component({
+  selector: 'go-mobile',
+  templateUrl: './go-mobile.component.html',
+  styleUrls: ['./go-mobile.component.scss']
+})
+export class GoMobileComponent implements OnInit {
+  sendingSMS = false;
+  saved = false;
+  profile = {
+    contactNumber: '',
+    email: '',
+    sendsms: true
+  };
+  invalidNumber = true;
+  // default country model
+  countryModel = 'AUS';
+  // default mask
+  mask: Array<string|RegExp>;
+  countryCodes: any[];
+
+  constructor(
+    private goMobileService: GoMobileService,
+    private utils: UtilsService,
+    private notification: NotificationService,
+    public storage: BrowserStorageService,
+    private contact: ContactNumberFormat,
+  ) {
+    this.countryCodes = contact.countryCodes;
```

Comment:
Why this comes back again?

Reply:
@shawnm0705 needs this bro
https://github.com/intersective/practera-app-v2/pull/318/files/70e284c09c7fe9dc47841273fd0fb89b7b5c79b7#diff-b900146e0fbc39787e4b5bd6bd80d534R40

Reply:
@trtshen We can use `contact.countryCodes` there as well